<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Control Gipper</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1181</width>
      <height>711</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>400</width>
       <height>250</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="btn_connect">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>120</width>
        <height>120</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 255, 0);</string>
      </property>
      <property name="text">
       <string>CONNECT</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_setting">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>150</y>
        <width>120</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 170, 255);</string>
      </property>
      <property name="text">
       <string>SETTING</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_io">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>200</y>
        <width>120</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 85, 0);</string>
      </property>
      <property name="text">
       <string>I/O</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>251</width>
        <height>231</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>6</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>10</y>
       <width>750</width>
       <height>331</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <widget class="QLabel" name="label_D">
      <property name="geometry">
       <rect>
        <x>29</x>
        <y>290</y>
        <width>60</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>00</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_D_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>20</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>V:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>80</width>
        <height>200</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Đồ thị 
vận tốc</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget_d" native="true">
      <property name="geometry">
       <rect>
        <x>99</x>
        <y>10</y>
        <width>640</width>
        <height>311</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="group_tuyChinh">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>400</width>
       <height>171</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
     </property>
     <property name="title">
      <string>Tùy Chỉnh giá trị D</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <widget class="QScrollBar" name="scroll_tuyChinh">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>375</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 255, 255);</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_gap">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>85</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 170, 255);</string>
      </property>
      <property name="text">
       <string>GẮP</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_pause">
      <property name="geometry">
       <rect>
        <x>105</x>
        <y>80</y>
        <width>85</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 170, 0);</string>
      </property>
      <property name="text">
       <string>PAUSE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_mo">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>80</y>
        <width>85</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 170, 255);</string>
      </property>
      <property name="text">
       <string>MỞ</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_reset">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>80</y>
        <width>85</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 170, 255);</string>
      </property>
      <property name="text">
       <string>RESET</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>680</y>
       <width>55</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_status">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>680</y>
       <width>500</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QFrame" name="frame_4">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>350</y>
       <width>750</width>
       <height>331</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <widget class="QLabel" name="label_I">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>290</y>
        <width>60</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>00</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_D_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>20</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>I:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>80</width>
        <height>200</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Đồ thị 
cường độ
dòng điện</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget_i" native="true">
      <property name="geometry">
       <rect>
        <x>99</x>
        <y>10</y>
        <width>640</width>
        <height>310</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="group_readWrite">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>450</y>
       <width>401</width>
       <height>231</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Read / Write</string>
     </property>
     <widget class="QPushButton" name="btn_send">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>50</y>
        <width>120</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>SEND</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_remove">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>170</y>
        <width>120</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>REMOVE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_add">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>110</y>
        <width>120</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ADD</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>80</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Options</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>80</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Value D</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>80</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Value I</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="t_edit_valueD">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>110</y>
        <width>120</width>
        <height>35</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="t_edit_valueI">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>170</y>
        <width>120</width>
        <height>35</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_options">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>120</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tahoma</family>
        <italic>true</italic>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_time">
     <property name="geometry">
      <rect>
       <x>1124</x>
       <y>680</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>1040</x>
       <y>680</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Time now:</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
